ssvsigner/go.mod

```diff
@@ -0,0 +1,93 @@
+module github.com/ssvlabs/ssv/ssvsigner
+
+go 1.22.6
+
+require (
+	github.com/alecthomas/kong v1.8.1
+	github.com/attestantio/go-eth2-client v0.24.1-0.20250212100859-648471aad7cc
+	github.com/carlmjohnson/requests v0.24.3
+	github.com/ethereum/go-ethereum v1.14.8
+	github.com/fasthttp/router v1.5.4
+	github.com/herumi/bls-eth-go-binary v1.29.1
+	github.com/ssvlabs/ssv v1.2.1-0.20250318213811-c9c1ec386275
```

Comment:
we using old version of ssv here. Newer versions (v2+) can't be imported. I know the crypto code probably haven't changed since then but we need to make sure we are not counting on the latest code to work in ssv-signer when using the libs. And in the future we need to fix importing ssv as a module.
@moshe-blox @nkryuchkov !!

Reply:
I used the specific commit and it's referenced here (`c9c1ec386275`). It wouldn't compile with an old version. I guess the v2+ issue is about this weirdly named version string: `v1.2.1-0.20250318213811-c9c1ec386275`